cli/src/test/kotlin/de/deutschepost/sdm/cdlib/archive/ArchiveCommandIntegrationTest.kt

```
@@ -171,9 +172,11 @@ class ArchiveCommandIntegrationTest(@Value("\${artifactory-azure-identity-token}
     override suspend fun beforeTest(testCase: TestCase) {
         super.beforeTest(testCase)
         val repository = artifactory.repository(repoName)
+        val logger = java.util.logging.Logger.getLogger(this::class.java.name)
         try {
             repository.delete(releaseName_build)
-        } catch (_: Exception) {
+        } catch (e: Exception) {
+            logger.warning("Failed to delete repository: ${e.message}")
```

To fix the issue with the empty catch block (SonarLint rule `kotlin:S108`), a logger is declared using `java.util.logging.Logger` since it was allowed and matches the standard requirement. The logger instance is created with the context of the current class. Within the catch block, a warning log statement is added to capture any exceptions that occur when deleting a repository in the `beforeTest` setup. This provides visibility into failures during test setup, assisting in debugging. The log level `warning` is appropriate as it highlights potential issues without failing the test suite entirely.<br><br>[Related Sonar Issue](http://localhost:9000/issues?open=15156638-4782-4aec-a774-3f852ff7c2ca)<br>[Related Sonar Rule](http://localhost:9000/coding_rules?open=kotlin:S108&rule_key=kotlin:S108)